### PR TITLE
LC Item functionality

### PR DESCRIPTION
**Pull request by lucyconklin**
Full functionality and passing tests for item. Not checked against spec harness. @case-eee

### CODE REVIEW

**Review by riverswb — APPROVED** [review]
Looks good to me, merging

File: app/controllers/api/v1/items/random_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Api::V1::Items::RandomController < ApplicationController
+
+  def show
+    render json: Item.order("RANDOM()").first
```
Comment (case-eee) [inline app/controllers/api/v1/items/random_controller.rb:4]: i'd suggest pulling this query out into a model method so we can potentially reuse it. this is a lot of logic for the controller 👍

File: lib/tasks/import.rake [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  desc "Import merchants from csv"
+  task merchants: :environment do
+    counter = 0
+    CSV.foreach("db/csv/merchants.csv", headers: true) do |row|
+      merchant = Merchant.create(id: row["id"],
+                                 name: row["name"],
+                                 created_at: row["created_at"],
+                                 updated_at: row["updated_at"] )
+      puts "#{name}- #{merchant.errors.full_messages.join(", ")}" if merchant.errors.any?
+      counter += 1 if merchant.persisted?
```
Comment (case-eee) [inline lib/tasks/import.rake:14]: like that you're making sure this is persisted 👍

**Review by case-eee — COMMENTED** [review]
@lucyconklin @riverswb 👍

**Pull request merged**